js/views/modals/Settings/Store.js

```diff
@@ -286,22 +286,11 @@ export default class extends baseVw {
           const remAvail = this.modsAvailable.removeModeratorsByID(this.modsAvailable.selectedIDs);
 
           this.modsByID.excludeIDs = this.currentMods;
-          this.modsByID.moderatorsStatus.setState({
-            hidden: true,
-          });
```

Comment:
These three sections of code were removed, they were just duplicating things already handled by the view and creating some weird edge case behavior (like if you loaded moderators, then closed and reopened settings the status bar would appear).